sources/application/TwigBase/Controller/Controller.php

```diff
@@ -437,8 +437,7 @@ public function DownloadZippedPage($aParams = array(), $sTemplateName = null)
 
 		if ($bFileTransfer)
 		{
-			header('Content-Description: File Transfer');
-			header('Content-Disposition: inline; filename="'.$sDownloadArchiveName);
+			header('Content-Disposition: attachment; filename="'.$sDownloadArchiveName);
```

Comment:
There is a missing '"' at the end (bug from previous version). Without it I have the same error on Safari (file named exec.php).

```suggestion
			header('Content-Disposition: attachment; filename="'.$sDownloadArchiveName.'"');
```

Tested on Safari & Firefox

Reply:
Good find. Didn't notice this and it seems the browsers didn't care anyway.